### PR TITLE
Add Polish translations for Jenkins plugin manager

### PR DESCRIPTION
**Pull request by damianszczepanik**
### Proposed changelog entries

* Adds Polish translations for plugin manager

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] (If applicable) Jira issue is well described - no JIRA issue
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). - no needed
- [x] Appropriate autotests or explanation to why this change has no tests - tested with local build
- [x] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [x] For dependency updates: links to external changelogs and, if possible, full diffs

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: core/src/main/resources/hudson/ProxyConfiguration/help-name_pl.html [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <div>
-  Je�li Jenkins znajduje si� za zapor� sieciow� i nie ma bezpo�redniego dost�pu do Internetu,
+  Jeśli Jenkins znajduje się za zaporą sieciową i nie ma bezpośredniego dostępu do Internetu,
```
Comment (damianszczepanik) [inline core/src/main/resources/hudson/ProxyConfiguration/help-name_pl.html:2]: Content is unchanged but encoding was updated from windows-1250 to utf-8

File: core/src/main/resources/hudson/PluginManager/installed_pl.properties [hunk 2]
```diff
@@ -30,13 +32,22 @@ Version=Wersja
...
 requires.restart=Wymagane jest ponowne uruchomienie Jenkinsa. Zmiany wtyczek w tym momencie s\u0105 bardzo niewskazane. Uruchom ponownie Jenkinsa, zanim wprowadzisz zmiany.
 Uninstallation\ pending=Trwa odinstalowywanie
+detached-disable=Wy\u0142\u0105czenie tej wtyczki mo\u017Ce nie by\u0107 bezpieczne
+This\ plugin\ cannot\ be\ enabled=Ta wtyczka nie mo\u017Ce by\u0107 w\u0142\u0105czona
 This\ plugin\ cannot\ be\ disabled=Ta wtyczka nie mo\u017Ce by\u0107 wy\u0142\u0105czona
+It\ has\ one\ or\ more\ enabled\ dependents=Ma jedn\u0105 lub wiele zainstalowanych zale\u017Cno\u015Bci
+This\ plugin\ cannot\ be\ uninstalled=Ta wtyczka nie mo\u017Ce by\u0107 odinstalowana
+It\ has\ one\ or\ more\ installed\ dependents=Ma jedn\u0105 lub wiele zainstalowanych zale\u017Cno\u015Bci
 No\ plugins\ installed.=Brak zainstalowanych wtyczek
-This\ plugin\ cannot\ be\ enabled=Ta wtyczka nie mo\u017Ce by\u0107 w\u0142\u0105czona
```
Comment (damianszczepanik) [inline core/src/main/resources/hudson/PluginManager/installed_pl.properties:35]: Moved above to keep together with other translations which reflect to same scope

**Review by oleg-nenashev — APPROVED** [review]
Looks legit. CC @ewelinawilkosz just in case

**Review by ewelinawilkosz — APPROVED** [review]
Looks Good! The only thing that doesn’t sit well with me is “Menedżer wtyczek”. @damianszczepanik would you consider „Zarządzaj wtyczkami”? 
It’s not a deal breaker for me though. I rarely use software in Polish and maybe “menedżer” is generally agreed way of translating the word.

**Comment by oleg-nenashev** [comment]
We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Comment by damianszczepanik** [comment]
I have translated to same label which is now on other tabs.
"Zarządzaj wtyczkami" would be nice for the button as this is an action.

**Pull request merged**